src/pages/projectCreate/components/ProjectFeeSelection.tsx

```diff
@@ -0,0 +1,97 @@
+import { Box, HStack, Text, useRadio, useRadioGroup, VStack } from '@chakra-ui/react'
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+
+type ProjectFeeSelectionProps = {
+  readOnly?: boolean
+  value: number
+  onChange: (value: number) => void
+}
+
+const ProjectFeeSelection: React.FC<ProjectFeeSelectionProps> = ({ readOnly, value, onChange }) => {
+  const { t } = useTranslation()
+
+  const options = [
+    { label: '0%', value: '0.0' },
+    { label: '2%', value: '0.02' },
+    { label: '4%', value: '0.04' },
+    { label: '6%', value: '0.06' },
+    { label: '8%', value: '0.08' },
+  ]
```

Comment:
You can have this static value, outside the component, that way, the variable wouldn't be initialized on every re-render.

Also might be a good idea to have these values in an enum, so that we could reference the number from there, everywhere else.

Reply:
Is there a specific file or folder where enums are defined?

Reply:
Not really, if it is just one then it can be in the same file just outside the component, if there are a few, you can create an extra file close to where it's used.